CMakeLists.txt

```diff
@@ -111,10 +112,9 @@ endif ()
 
 
 # Set the default namespace
-if (OSL_NAMESPACE)
-    add_definitions ("-DOSL_NAMESPACE=${OSL_NAMESPACE}")
-endif ()
-message(STATUS "Setting Namespace to: ${OSL_NAMESPACE}")
+set (PROJ_NAMESPACE "${${PROJ_NAME}_NAMESPACE}")
```

Comment:
Oh, oops, let me take a look. I may need to add that definition, as it's part of things that recently changed in master.